backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -216,14 +216,18 @@ void DumpWithoutCrash(jstring message) {
             annotations = new crashpad::SimpleStringDictionary();
             info->set_simple_annotations(annotations);
         }
-
-        // user can't override error.message - exception message that Crashpad/crash-reporting tool
-        // will set to tell user about error message. This code will set error.message only for single
-        // report and after creating a dump, method will clean up this attribute.
-        jboolean isCopy;
-        const char *rawMessage = env->GetStringUTFChars(message, &isCopy);
-        annotations->SetKeyValue("error.message", rawMessage);
-        env->ReleaseStringUTFChars(message, rawMessage);
+        if(set_main_thread_as_faulting_thread == true) {
+            annotations->SetKeyValue("_mod_faulting_tid", thread_id);
+        }
+        if(message != NULL) {
```

Comment:
Why do we repeat this `message != NULL` check? Maybe we can fix this now or later

Reply:
I did that for one purpose  - if user didn't define message and user don't want to change faulting thread to main thread - there is no need to read Crashpad attributes and set them.

Reply:
Ah okay that makes sense